dex/pair/src/lib.rs

```diff
@@ -99,10 +100,16 @@ pub trait Pair<ContractReader>:
             );
             self.add_permissions_for_all(admins, Permissions::ADMIN);
         };
+
+        self.safe_price_round_save_interval()
+            .set_if_empty(DEFAULT_ROUND_SAVE_INTERVAL);
```

Comment:
It's always empty on init.

```suggestion
        self.safe_price_round_save_interval()
            .set(DEFAULT_ROUND_SAVE_INTERVAL);
```

Reply:
Done.